src/components/hoc/with-moviestore-service.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { MoviesServiceConsumer } from '../movies-service-context';
+
+const withMovieStoreService = () => (Wrapped) => {
```

Comment:
do you really need to use carrying here? I mean do you need wrapping function without args